deploy/yamnet-detector/app.py

```
@@ -0,0 +1,107 @@
+
+
+app = FastAPI(title="Canticle YAMNet instrumental classifier", lifespan=lifespan)
+
+
+@app.get("/health")
+def health():
+    return {"status": "ok", "classes": len(_state.get("classes", []))}
+
+
+@app.post("/classify")
+async def classify(file: UploadFile = File(...)):
```

Ruff (0.15.18)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)